package.json

```diff
@@ -89,6 +89,7 @@
     "react-reactive-form": "1.0.27",
     "react-redux": "^7.2.2",
     "react-router-dom": "^5.2.0",
+    "react-select": "^3.1.1",
```

Comment:
Why do we need this package? It increases the bundle by 27k zipped! Let's find another solution that is not that expensive. We have an autosuggest component, maybe you can use that one?

Reply:
Hmm, that's unfortunate.
Where can I find your autosuggest component?

Reply:
Done. Updated the initial message of this PR at the top of this page.